### DISCUSSION TITLE
Custom Avatar Animations

### DISCUSSION THREAD

**Discussion by futurestages** [mozilla/hubs#3030] [General Questions & Discussion] [answered]
I'm trying to add my own trigger animations to the default avatar. I can get animation to trigger using X, Y, and Thumbstick-L...but any pressure on the button triggers, so I tried to assign animation to Pinch-L (or mrpDown_L). But as soon as I press the pinchL with this customization, it crashes Mozilla? 

Any thoughts?
![export_settings](https://user-images.githubusercontent.com/70774583/93249880-d31ebd80-f746-11ea-831f-f5f1f6fe9812.PNG)
![point_L](https://user-images.githubusercontent.com/70774583/93249883-d44fea80-f746-11ea-8649-17189240b8cd.PNG)
![thumbDown_L](https://user-images.githubusercontent.com/70774583/93249891-d5811780-f746-11ea-9bc2-ba5045e9c9a9.PNG)

**Comment by emclaren** [comment, marked as answer]
can you tell us a little more about what happens when it crashes? (e.g. Does the whole browser stop, or does the room freeze)? Does this happen in different browsers?

If you are able to open the browser developer tools and let us know if there are any red errors at the bottom of the console output it would be helpful. 

To do this, If you open this room on chrome or firefox, and then open the developer console (press command + option + i on mac, let me know if you need PC instructions). Then select Console from the top of the popup that appears. If you could scroll to the bottom and can send us a screenshot / copy and paste the text from the console or save the text to a file (See screenshot) it will give us an indication what is going wrong. 

<img width="682" alt="Screen Shot 2020-09-08 at 10 13 59 PM" src="https://user-images.githubusercontent.com/6835324/93520638-a653dd00-f8e3-11ea-8c2e-1abcb0920420.png">

**Reply by emclaren** [reply]
Hi there, I'm going to mark this question as answered for the moment since we haven't heard any more information, but if the problem persists please feel free to respond and let us know!
